prometheus-ksonnet/lib/prometheus-ha-mixin.libsonnet

```diff
@@ -1,104 +1,8 @@
-local k = import 'ksonnet-util/kausal.libsonnet';
-local container = k.core.v1.container;
-local statefulset = k.apps.v1.statefulSet;
-local configMap = k.core.v1.configMap;
-
+(import 'prometheus/ha-mixin.libsonnet') +
```

Comment:
I wonder if it might not make sense to `git mv prometheus-ksonnet/lib/prometheus-ha-mixin.libsonnet prometheus/ha-mixin.libsonnet`, thereby maintaining all the history of quite a fundamental file - and then creating a new file in `prometheus-ksonnet/lib/prometheus-ha-mixin.libsonnet` with the current contents.

Reply:
I think this gets lost in the squash merge anyway. :neutral_face:

Reply:
Don't think I follow.
If you `git mv`, the path of the file may change but its history will be maintained. Squashing will combine all your changes in this PR only.